lib/tmuxinator/project.rb

```diff
@@ -56,14 +79,6 @@ def pre_window
       end
     end
 
-    def attach
-      attach = true
-      if !yaml["attach"].nil?
```

Comment:
@dg-ratiodata You're totally right! I'd assume it can actually be `false` so my advice is just plain wrong. :flushed: